spec/fixtures/.release

```diff
@@ -1,8 +1,6 @@
 ---
-addons:
```

Comment:
This is actually a valid `.release` file, as can be written by the ruby buildpack. Is there a reason why you removed it?